ceph-ansible-pull-requests/build/teardown

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
+# There has to be a better way to do this than this script which just looks
```

Comment:
Actually, looks like ``tox`` won't be a great solution for this either as I can't have one environment with multiple ``changedirs``.